docs/guides/qedma-qesem.ipynb

```diff
@@ -143,143 +167,171 @@
+    "\n",
+    "\n",
+    "- `transpilation_level`: After a circuit is submitted to QESEM, it automatically prepares several alternative circuit transpilations and chooses the one that minimizes QPU time. For instance, alternative transpilations might utilize Qedma-optimized fractional RZZ gates to reduce the circuit depth. Of course, all transpilations are equivalent to the input circuit, in terms of their ideal output. To exert more control over the circuit transpilation, set the transpilation level in the `options`. While `\"transpilation_level\": 1 ` corresponds to the default behavior described above, `\"transpilation_level\": 0 ` includes only minimal modifications required for high-accuracy output; for example, ‘layerification’ - the organization of circuit operations into ‘layers’ of simultaneous two-qubit gates. Note that automatic hardware-mapping onto high-fidelity qubits is applied in any case.\n",
+    "\n",
+    "| `transpilation_level` | description |\n",
+    "|:-:|:--|\n",
+    "| `1` | Default QESEM transpilation. Prepares several alternative transpilations and chooses the one that minimizes QPU time. Barriers may be modified in the layerification step. |\n",
+    "| `0` | Minimal transpilation: the mitigated circuit will closely resemble the input circuit structurally. Circuits provided in level 0 should match the device connectivity and should be specified in terms of the following gates: CX, Rzz(α), and standard single-qubit gates (U, x, sx, rz, etc). Barriers will be respected in the layerification step. |\n",
+    "\n",
+    "<Admonition type=\"note\">\n",
+    "Qiskit barriers are typically used to specify the layers of two-qubit gates in quantum circuits. In level 0, QESEM preserves the layers specified by the barriers. In level 1, the layers specified by the barriers are considered as one transpilation alternative when minimizing QPU time.\n",
```

Comment:
I think "Qiskit barriers" feels odd? This should read "In Qiskit, barriers"? 

CC: @abbycross

Reply:
Let me know how you recommend to phrase it.

Reply:
How about "Barrier operations are typically used..." (just leave off Qiskit altogether)?